src/main/java/org/spongepowered/api/item/inventory/ItemStackSnapshot.java

```diff
@@ -57,6 +57,13 @@
     int getCount();
 
     /**
+     * Returns true if this ItemStackSnapshot is of Type air or has 0 or less quantity.
```

Comment:
Don't mention that it is of type air.

Simply mention that the implementation defines what empty means.